lib/Basics/debugging.h

```diff
@@ -146,6 +146,11 @@ struct remove_cvref {
 template<class T>
 using remove_cvref_t = typename remove_cvref<T>::type;
 
+template<typename>
+struct is_basic_string : std::false_type {};
+template<typename C, typename T, typename A>
+struct is_basic_string<std::basic_string<C, T, A>> : std::true_type {};
```

Comment:
Maybe we should check only char string?

Reply:
Discussed with @MBkkt: can be ignored for now.